CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
+ - Fix: Read of undefined property if cluster is offline.
```

Comment:
Fix: Reading an undefined property while cluster is offline caused Javascript error